### PR TITLE
Gamestats Initial Methods

### PR DESCRIPTION
**Pull request by kbug819**
## Description

Please include a description of what was changed:
Includes initial methods and testing for game statistics class and spec.

Refactored class and updated testing and class setup per group standards.
Removed .gitignore data to match main to avoid conflicts.

## Type of change

- [x] fix
- [x] feat
- [x] test
- [x] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

**Review by ethanb1145 — COMMENTED** [review]
I like that you implemented a helper method to make the code a bit easier. Code looks easy to follow and is well organized, good job.

File: spec/game_statistics_spec.rb [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+    end
+  end
+
+  describe "#percent ties" do
+    it "finds percntage of tied away and home games" do
+      expect(@game_stats.percentage_ties).to eq(0.05)
+    end
+  end
+
+  describe "#percentage_calculator" do
```
Comment (elizakeating) [inline spec/game_statistics_spec.rb:54]: Nice testing of the helper method!

**Comment by elizakeating** [comment]
Code looks super clean! The helper method was a smart idea! Nice work!!

**Comment by nurudeensulaimon** [comment]
Reviewed.  Thanks for making your codes clear and concise

**Pull request merged**